[FEATURE] Generalize DataProcessor #27 (#28)
b13/container: README.md

### TypoScript

>     // default/general configuration (will add 'children_<colPos>' Variable to processedData foreach colPos in container

Reviewer: ```suggestion
    // default/general configuration (will add 'children_<colPos>' variable to processedData for each colPos in container
```

    tt_content.b13-2cols-with-header-container < lib.contentElement
    tt_content.b13-2cols-with-header-container {
        templateName = 2ColsWithHeader
        templateRootPaths {
            10 = EXT:container/Resources/Private/Contenttypes
        }
        dataProcessing {
            100 = B13\Container\DataProcessing\ContainerProcessor
        }
    }

>     // explicite by colPos

Reviewer: ```suggestion
    // if need be you can use ContainerProcessor with explicitly set colPos/variable values
```

### Template

    <f:for each="{children_100}" as="record">

>         {record.header} <br />

Reviewer: ```suggestion
        {record.header} <br>
```

        <f:format.raw>
            {record.renderedContent}
        </f:format.raw>
    </f:for>

    <f:for each="{children_101}" as="record">

>         {record.header} <br />

Reviewer: ```suggestion
        {record.header} <br>
```

        <f:format.raw>
            {record.renderedContent}
        </f:format.raw>
    </f:for>

> with explicite colPos defined use children<Left|Right> analog

Reviewer: ```suggestion
with explicit colPos defined use '{children<Left|Right>}' as set in the example above
```

## Concepts
- Complete Registration is done with one PHP call to TCA Registry

> - A container in the BE Page-Module is rendered like a page itselfs (s. View/ContainerLayoutView)

Reviewer: ```suggestion
- A container in the BE Page-Module is rendered like a page itself (s. View/ContainerLayoutView)
```
